planex/cmd/createmockconfig.py

```diff
@@ -0,0 +1,123 @@
+"""
+planex-create-mock-config: Populate mock config with yum repositories
+"""
+
+from __future__ import print_function
+import argparse
+import os
+import os.path
+import pprint
+import sys
+import yum
+
+import ConfigParser
+import StringIO
+
+import argcomplete
+
+from planex.util import add_common_parser_options
+from planex.util import setup_logging
+from planex.util import setup_sigint_handler
+
+
+def load_mock_reference(fname):
+    """
+    read in the reference mock configuration
```

Comment:
I don't think so, given that what mock itself does is similar.   I would just like a comment to reassure a future reviewer that it was done this way after careful consideration of the alternatives! :)